src/style.css

```diff
@@ -2,2 +1,85 @@
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+li,
+a {
+  text-decoration: none;
+}
```

Comment:
It's not a good practice to use tag selectors for styling. Use class selectors instead. It will make your styles more flexible and prevent unexpected issues in the future.